HdcpProfile/HdcpProfile.cpp

```diff
@@ -126,7 +126,7 @@ namespace WPEFramework
                 
                 _hdcpProfile->Unregister(&_hdcpProfileNotification);
                 Exchange::JHdcpProfile::Unregister(*this);
-
+		configure->Release();
```

Comment:
Does it have to be released in Deinitialize?
Can you release immediately after Configure (Ex: https://github.com/rdkcentral/entservices-softwareupdate/blob/develop/FirmwareUpdate/FirmwareUpdate.cpp#L83)?